memcached.c

```diff
@@ -9015,10 +9047,12 @@ int main (int argc, char **argv) {
             exit(EX_USAGE);
 #endif
             break;
+#ifndef DISABLE_UNIX_SOCKET
```

Comment:
compilers may complain about switch coverage... could move the ifdef's inside the parser and print a text error about it being disabled.